### PR TITLE
[docs] Update article about switching EE to CE and vice versa for 1.65 

### PR DESCRIPTION
**Pull request by sprait**
## Description

- Add changes from PR #9792 to main branch.
- Add a step to manually pull the REGISTRY_PACKAGE_PROXY image.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Update article about switching EE to CE.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -854,63 +854,275 @@ The instruction implies using the public address of the container registry: `reg
 Deckhouse CE does not support cloud clusters on OpenStack and VMware vSphere.
 {% endalert %}
 
-To switch Deckhouse Enterprise Edition to Community Edition, follow these steps:
+Perform the following steps to switch a Deckhouse Enterprise Edition cluster to Community Edition (all commands must be run on the master node of the active cluster):
 
-1. Make sure that the modules used in the cluster [are supported in Deckhouse CE](revision-comparison.html). Disable modules that are not supported in Deckhouse CE.
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:859]: @sprait Why we deleting the phrase?

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-FAQ.md:859]: It looks like this phrase should be deleted in both languages... ?

**Review by z9r5 — APPROVED** [review]

**Pull request merged**